src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8134,6 +8140,9 @@ private StatementSyntax ParseLocalDeclarationStatement(SyntaxToken awaitKeywordO
             }
             bool canParseAsLocalFunction = usingKeyword == null;
 
+            var attributes = _pool.Allocate<AttributeListSyntax>();
+            ParseAttributeDeclarations(attributes);
```

Comment:
should have a feature-flag/version-check right?

Reply:
Yes, thanks. Will make sure to `// PROTOTYPE` this.

Reply:
I went ahead and added the feature flag check for this scenario.

Reply:
Did you add tests for the language feature flag?

---
In reply to: [328751045](https://github.com/dotnet/roslyn/pull/38808#discussion_r328751045) [](ancestors = 328751045)